scripts/src/main/resources/scripts/devon

```diff
@@ -72,24 +72,50 @@ then
   mkdir -p ~/.devon  
   echo "Copying devon CLI script to your home directory..."
   cp "${BASH_SOURCE:-$0}" ~/.devon/devon
+  AUTOCOMPLETION="source ${DEVON_IDE_HOME}/scripts/devon-autocomplete"
+  
   if [ ! -e ~/.bashrc ]
   then
     touch ~/.bashrc
   fi
   if ! grep -q "alias devon=" ~/.bashrc
   then
     echo "Installing devon alias to your .bashrc"
-    echo -e '\nalias devon="source ~/.devon/devon"\ndevon' >> ~/.bashrc
+    {
+      echo -e '\nalias devon="source ~/.devon/devon"\ndevon'
+      echo -e "${AUTOCOMPLETION}"
+    } >> ~/.bashrc
   fi
+
   if [ ! -e ~/.zshrc ]
   then
     touch ~/.zshrc
   fi
   if ! grep -q "alias devon=" ~/.zshrc
   then
     echo "Installing devon alias to your .zshrc"
-    echo -e '\nalias devon="source ~/.devon/devon"\ndevon' >> ~/.zshrc
+    {
+      echo -e '\nalias devon="source ~/.devon/devon"\ndevon'
+      echo -e "autoload -Uz compinit\ncompinit\nautoload bashcompinit\nbashcompinit\n${AUTOCOMPLETION}"
```

Comment:
Very nice but I have some questions:
* what if the user already has some manual autocompletion in his shell config (`.zshrc`)? Will duplicate invocation of `compinit` cause any harm? Is that always available on all platforms as it is a build-in command of both `bash` and `zsh`?
* what if the user already has devonfw-ide installed? Should we offer a central command to install or uninstall the autocompletion in `ide` commandlet instead and invoke that from the existing places?

Reply:
- You are right, I have not tested this case. There seems to be problems when compinit already exists. I will check if compinit or bashcompinit exists before adding the config. I tested this on the Mac and didn't have to install anything to do it. I will test it on Linux later, but think it will work.
- If `devon-ide` is installed, `devon-ide` must be reinstalled. Alternatively I can extend the `devon ide update` command that it extends the configuration. Basically I think that it is not useful to have an extra command to install the autocomplete, because actually every user finds this useful and can use it. After all, there are no disadvantages to having autocompletion.

Reply:
I tested it with WSL Ubuntu and zsh with autocomplete works fine.

Reply:
> Basically I think that it is not useful to have an extra command to install the autocomplete, because actually every user finds this useful and can use it.

My intention was not to make it optional but just to have the ability to "reinstall" it e.g. if the initial installation failed, has been lost due to changes in `.bashrc` or `.zshrc`, etc.